test/test_car_parser.py

```diff
@@ -178,7 +178,7 @@ def test_car_parser(self):
             self.assertTrue('StartTime' in cont)
             self.assertTrue('EndTime' in cont)
 
-            for key in cases[car].keys():
+            for key in list(cases[car].keys()):
```

Comment:
Superfluous `list(….keys()` here.
```suggestion
            for key in cases[car]:
```

Reply:
Noted in #356